src/api/xinput/XInputDLL.cpp

```diff
@@ -47,16 +48,16 @@ bool CXInputDLL::Load(void)
 
   m_strVersion = "1.4";
   dsyslog("Attempting to load XInput1_4.dll...");
-  m_dll = LoadLibrary("XInput1_4.dll");  // 1.4 Ships with Windows 8
+  m_dll = LoadLibrary(KODI::PLATFORM::WINDOWS::ToW("XInput1_4.dll").c_str());  // 1.4 Ships with Windows 8
```

Comment:
Easier to just add an L before the "", no conversion necessary. If you want compatibility with both versions you can use the T() macro instead